src/ImageRunModal.jsx

```diff
@@ -32,135 +38,177 @@ const units = {
+                                <OutlinedQuestionCircleIcon />
+                            </button>
+                        </Popover>
+                    }>
+                    <TextInput id={id + "-ip-address"}
+                            value={item.IP || ''}
+                            onChange={value => onChange(idx, 'IP', value)} />
+                </FormGroup>
+                <FormGroup className="pf-m-2-col-on-sm"
+                        label={_("Host port")}
+                        fieldId={id + "-host-port"}
+                        labelIcon={
+                            <Popover aria-label={_("Host port help")}
+                                enableFlip
+                                bodyContent={_("If the host port is not set the container port will be randomly assigned a port on the host.")}>
+                                <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                                    <OutlinedQuestionCircleIcon />
+                                </button>
+                            </Popover>
+                        }>
+                    <TextInput id={id + "-host-port"}
+                                value={item.hostPort || ''}
+                                onChange={value => onChange(idx, 'hostPort', value)} />
+                </FormGroup>
+                <FormGroup className="pf-m-3-col-on-sm"
+                            label={_("Container port")}
+                            fieldId={id + "-container-port"} isRequired>
+                    <TextInput id={id + "-container-port"}
+                                value={item.containerPort || ''}
+                                onChange={value => onChange(idx, 'containerPort', value)} />
+                </FormGroup>
+                <FormGroup className="pf-m-2-col-on-sm"
+                            label={_("Protocol")}
+                            fieldId={id + "-protocol"}>
+                    <FormSelect className='pf-c-form-control container-port-protocol'
+                                id={id + "-protocol"}
+                                value={item.protocol}
+                                onChange={value => onChange(idx, 'protocol', value)}>
+                        <FormSelectOption value='tcp' key='tcp' label={_("TCP")} />
+                        <FormSelectOption value='udp' key='udp' label={_("UDP")} />
+                    </FormSelect>
+                </FormGroup>
+                <FormGroup className="pf-m-1-col-on-sm remove-button-group">
+                    <Button variant='secondary'
+                                className="btn-close"
+                                id={id + "-btn-close"}
+                                isSmall
+                                aria-label={_("Remove item")}
+                                icon={<MinusIcon />}
+                                onClick={() => removeitem(idx)} />
+                </FormGroup>
```

Comment:
This button should be red

Reply:
I actually don't agree with the design on this. Red buttons imply dangerous operations, that can't be reverted etc. Removing options in the dialog does not fall into this category.
@garrett What do you think?